src/content/docs/en/pages/devtools/cli/azion-cli-alpha/rules-engine/rules-engine.mdx

```diff
@@ -1,12 +1,16 @@
 ---
-title: Rules engine
+title: Azion CLI rules-engine
 description: Create, list, describe, update, and delete your application's Rules Engine's rules through Azion CLI. 
 meta_tags: cli,edge computing, rules engine, low code
 namespace: documentation_cli_rules_engine
 menu_namespace: cliMenuAlpha
 permalink: /documentation/products/cli/rules-engine-crud/
 ---
 
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case any required field isn't informed through the specific flags, an interactive message prompts and asks for the missing information.
```